### PR TITLE
Remove 6.2.3 as a duplicate

### PR DESCRIPTION
**Pull request by tghosth**
This Pull Request relates to issue #1198

### CODE REVIEW

**Comment by tghosth** [comment]
@jmanico any further comments or questions? See my notes here: https://github.com/OWASP/ASVS/issues/1198#issuecomment-1358924019

**Comment by tghosth** [comment]
@set-reminder 2 weeks merge if no adverse feedbac

File: 5.0/en/0x14-V6-Cryptography.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Although this section is not easily penetration tested, developers should consid
 | :---: | :--- | :---: | :---: | :---: | :---: |
 | **6.2.1** | Verify that all cryptographic modules fail securely, and errors are handled in a way that does not enable Padding Oracle attacks. | ✓ | ✓ | ✓ | 310 |
 | **6.2.2** | Verify that industry proven or government approved cryptographic algorithms, modes, and libraries are used, instead of custom coded cryptography. ([C8](https://owasp.org/www-project-proactive-controls/#div-numbering)) | | ✓ | ✓ | 327 |
-| **6.2.3** | Verify that encryption initialization vector, cipher configuration, and block modes are configured securely using the latest advice. | | ✓ | ✓ | 326 |
+| **6.2.3** | [DELETED] | | | | |
```
Comment (elarlang) [inline 5.0/en/0x14-V6-Cryptography.md:31]: `[DELETED, DUPLICATE OF 6.2.5]`

**Review by elarlang — CHANGES_REQUESTED** [review]

File: 5.0/en/0x14-V6-Cryptography.md (hunk 1 above)
Comment (tghosth) [reply 5.0/en/0x14-V6-Cryptography.md]: Fixed

**Review by elarlang — APPROVED** [review]

**Review by jmanico — APPROVED** [review]

**Pull request merged**